safari/Refined GitHub.xcodeproj/project.pbxproj

```diff
@@ -231,13 +276,30 @@
 			path = "macOS (Extension)";
 			sourceTree = "<group>";
 		};
-		E35EB50526F36E0A00DDEA60 /* Resources */ = {
+		E3919AC127E6107E009C4956 /* Resources */ = {
 			isa = PBXGroup;
 			children = (
+				E3919AEA27E61192009C4956 /* background.js */,
+				E3919AF327E61192009C4956 /* background.js.map */,
+				E3919AEB27E61192009C4956 /* browser-polyfill.js */,
+				E3919AED27E61192009C4956 /* icon.png */,
+				E3919AE727E61192009C4956 /* manifest.json */,
+				E3919AF627E61192009C4956 /* options.css */,
+				E3919AF527E61192009C4956 /* options.css.map */,
```

Comment:
@fregante Should the source map files be included?

Reply:
I'll test this later and let you know. I forgot what we decided

Reply:
It looks like we include them:

<img width="372" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/159152064-35bc1b9a-795b-46b6-a82e-f6cf9c885459.png">